archlinux/PKGBUILD.qubes-post-upgrade.hook

```diff
@@ -0,0 +1,10 @@
+[Trigger]
+Operation = Upgrade
+Type = Path
+Target = *
+
+[Action]
+Description = Notifying dom0 about successful upgrade...
+Depends = qubes-vm-qrexec
+When = PostTransaction
+Exec = /usr/lib/qubes/qrexec-client-vm dom0 qubes.NotifyUpdates /bin/sh -c 'echo 0'
```

Comment:
Shouldn't this check if actually all updates got applied (like, if it wasn't update of a single package out of several updates available)?
For example, if I do `pacman -S xterm` (having xterm installed but at older version), it clears the updates available flag, even if several other updates are available.
Calling `/usr/lib/qubes/upgrades-status-notify` instead of this command should be enough.